src/components/ZclDomainClusterView.vue

```diff
@@ -454,6 +456,15 @@ export default {
       ],
     }
   },
+  created() {
+    if (this.clusters !== undefined) {
+      this.clusters.map((cluster) => {
+        if (cluster.domainName === 'General' && cluster.name === 'Basic') {
```

Comment:
Is this tutorial only for Zigbee pro? Do we have a separate one for Matter?
I see references for "General" and "Basic"